### PR TITLE
Fix storybook

### PR DESCRIPTION
**Pull request by kfirazu**
App responsiveness
stories in storybook

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,15 +8,19 @@ import SortBar from './components/Sort-Bar/sort-bar'
 import FeedList from './components/FeedList/feed-list'
 import newsData from './data/news.json'
 import Dashboard from './components/Dashboard/dashboard'
-import { newsService } from './services/news.service'
+// import { newsService } from './services/news.service'
 // import { filterByInterface } from './models/filter-by-interface'
```
Comment (orrosmanhls) [inline src/App.tsx:12]: remove comments

File: src/components/App-Header/mobile-header.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            {isMobile &&
+                <IconContainer children={[searchIcon, settingsIcon, notificationIcon]} />
+            }
+            {isTablet && !isMobile &&
+                
+                    <StyledInputContainer>
+                        <CustomInput name='search' id='search' placeholder='Search' handleChange={handleChange} handleFocus={handleFocus} label={'Text'} />
+                        {/* {isFocused &&
+                            <RecentSearchDropdown isFocused={isFocused} onCloseModal={onCloseModal} />
+                        } */}
```
Comment (orrosmanhls) [inline src/components/App-Header/mobile-header.tsx]: remove

File: src/components/App-Header/mobile-header.tsx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+const MobileHeader = () => {
+
+    const isMobile = useIsMobile()
+    const isTablet = UseIsTablet()
+
+    const [isSearchOpen, setIsSearchOpen] = useState<boolean>(false)
+    const [isSettingsOpen, setisSettingsOpen] = useState<boolean>(false)
+    const [isNotificationOpen, setisNotificationOpen] = useState<boolean>(false)
+    const [isFocused, setIsFocused] = useState<boolean>(false)
```
Comment (orrosmanhls) [inline src/components/App-Header/mobile-header.tsx:21]: `isFocused` is not used

File: src/components/App-Header/mobile-header.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+    const handleFocus = () => {
+        setIsFocused(true)
+    }
+
+    const handleChange = () => { console.log('clicked') }
+
+    const onCloseModal = () => {
+        setIsFocused(false)
+    }
```
Comment (orrosmanhls) [inline src/components/App-Header/mobile-header.tsx:32]: remove - `isFocused` not used

File: src/components/Arrow-Right-Icon/arrow-right.tsx [hunk 5]
```diff
@@ -1,15 +1,12 @@
...
+import  { StyledSvg } from "./arrow-right.style"
 
 const ArrowRightIcon = () => {
     return (
-        <StyledArrowRight>
-            <StyledSvg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
-                <path d="M15.9998 6.99991L0.999756 6.99991" stroke="white" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
-                <path d="M9.94995 0.975506L16 6.99951L9.94995 13.0245" stroke="white" strokeWidth="1.5" strokeLinecap="round" strokeLinejoin="round" />
-            </StyledSvg>
+        <StyledSvg width="17" height="14" viewBox="0 0 17 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (orrosmanhls) [inline src/components/Arrow-Right-Icon/arrow-right.tsx:5]: extract svg to asset

File: src/components/Article-Preview/article-preview.style.ts
Comment (orrosmanhls) [inline src/components/Article-Preview/article-preview.style.ts:1]: remove comments

File: src/components/Dropdown/custom-dropdown.stories.ts [hunk 6]
```diff
@@ -1,30 +1,43 @@
 import { Meta, StoryObj } from "@storybook/react";
 import CustomDropdown from "./custom-dropdown";
-import { MenuItem } from "@mui/material";
+import { StyledMenuListSX } from "./custom-dropdown.styles";
+// import * as MenuItem from './menu-item.stories'
```
Comment (orrosmanhls) [inline src/components/Dropdown/custom-dropdown.stories.ts]: remove

File: src/components/Dropdown/custom-dropdown.stories.ts [hunk 7]
```diff
@@ -1,30 +1,43 @@
...
+        items: mockCategories,
+        sx: StyledMenuListSX
+
     }
 }
 
+// export const WithArgs: Story = {
+//     render: (args: any) => <CustomDropdown {...args} />
+// }
+
```
Comment (orrosmanhls) [inline src/components/Dropdown/custom-dropdown.stories.ts]: remove

File: src/components/Dropdown/custom-dropdown.tsx [hunk 8]
```diff
@@ -35,8 +35,9 @@ const CustomDropdown: FC<CustomDropdownProps> = (props) => {
                     onClick={toggleDropdown}
                     IconComponent={() => null}
                     id={id}
-                    name={id}
+                    name={name}
                     value={selectedOption}
+                    // items={items}
```
Comment (orrosmanhls) [inline src/components/Dropdown/custom-dropdown.tsx]: remove

File: src/components/Input/custom-input.tsx [hunk 9]
```diff
@@ -10,14 +12,17 @@ export interface CustomInputProps {
     id: string
     handleChange: () => void
     handleFocus: () => void
-    handleBlur: (ev:FocusEvent<HTMLInputElement>) => void
+    // handleBlur?: (ev: FocusEvent<HTMLInputElement>) => void
```
Comment (orrosmanhls) [inline src/components/Input/custom-input.tsx]: remove

File: src/App.tsx (hunk 1 above)
Comment (orrosmanhls) [inline src/App.tsx:12]: remove

**Review by orrosmanhls — CHANGES_REQUESTED** [review]

File: src/components/Dropdown/custom-dropdown.tsx (hunk 8 above)
Comment (kfirazu) [reply src/components/Dropdown/custom-dropdown.tsx]: Done

File: src/components/Arrow-Right-Icon/arrow-right.tsx (hunk 5 above)
Comment (kfirazu) [reply src/components/Arrow-Right-Icon/arrow-right.tsx:5]: resolved already in another pr - feature-side bar

**Review by orrosmanhls — APPROVED** [review]

**Pull request merged**